prospero-common/src/main/java/org/wildfly/prospero/actions/UpdateAction.java

```diff
@@ -116,6 +117,8 @@ public boolean buildUpdate(Path targetDir) throws ProvisioningException, Operati
 
             final boolean result = prepareCandidateAction.buildCandidate(targetDir, galleonEnv,
                     ApplyCandidateAction.Type.UPDATE, provisioningConfig);
+
+            prepareCandidateAction.writeChannelMapToFile(updateSet, installDir);
```

Comment:
that should be written to the generated candidate (`targetDir`) not the original installation